### PR TITLE
Error message for disabling random write in block cache

### PR DESCRIPTION
**Pull request by souravgupta-msft**

### CODE REVIEW

**Review by vibhansa-msft — APPROVED** [review]

File: common/types.go [hunk 1]
```diff
@@ -70,6 +70,8 @@ const (
 	FuseAllowedFlags = "invalid FUSE options. Allowed FUSE configurations are: `-o attr_timeout=TIMEOUT`, `-o negative_timeout=TIMEOUT`, `-o entry_timeout=TIMEOUT` `-o allow_other`, `-o allow_root`, `-o umask=PERMISSIONS -o default_permissions`, `-o ro`"
 
 	UserAgentHeader = "User-Agent"
+
+	BlockCacheRWErrMsg = "Due to data integrity issues, your random write flow using block cache is blocked. Please reach out to blobfusedev@microsoft.com or create a GitHub issue if you hit this issue. We are working on fixing this and enabling it back. For more details, refer https://aka.ms/blobfuse2warnings."
```
Comment (tjvishnu) [inline common/types.go:74]: Proposed message: "**Notice:** The random write flow using block cache is temporarily blocked due to potential data integrity issues. This is a precautionary measure. If you see this message, please contact [blobfusedev@microsoft.com](mailto:blobfusedev@microsoft.com) or create a GitHub issue. We're working on a fix. More details: [https://aka.ms/blobfuse2warnings](https://aka.ms/blobfuse2warnings)."

Comment (souravgupta-msft) [reply common/types.go]: Updated

File: common/types.go [hunk 2]
```diff
@@ -70,6 +70,8 @@ const (
 	FuseAllowedFlags = "invalid FUSE options. Allowed FUSE configurations are: `-o attr_timeout=TIMEOUT`, `-o negative_timeout=TIMEOUT`, `-o entry_timeout=TIMEOUT` `-o allow_other`, `-o allow_root`, `-o umask=PERMISSIONS -o default_permissions`, `-o ro`"
 
 	UserAgentHeader = "User-Agent"
+
+	BlockCacheRWErrMsg = "Notice: The random write flow using block cache is temporarily blocked due to potential data integrity issues. This is a precautionary measure. If you see this message, contact blobfusedev@microsoft.com or create a GitHub issue. We're working on a fix. More details: https://aka.ms/blobfuse2warnings."
```
Comment (vibhansa-msft) [inline common/types.go:74]: Shall we add a new line before "if you see this message" just for more clarity

Comment (souravgupta-msft) [reply common/types.go]: Added

**Review by ashruti-msft — APPROVED** [review]

**Pull request merged**